movement/movement.h

```diff
@@ -109,13 +110,16 @@ typedef enum {
     EVENT_TIMEOUT,              // Your watch face has been inactive for a while. You may want to resign, depending on your watch face's intended use case.
     EVENT_LIGHT_BUTTON_DOWN,    // The light button has been pressed, but not yet released.
     EVENT_LIGHT_BUTTON_UP,      // The light button was pressed and released.
```

Comment:
Perhaps comments on this should change, as AFAIK this event doesn't occur after a long press.

Ideally, it would be renamed EVENT_LIGHT_BUTTON_SHORT_UP I think (i.e. globally replace this in the codebase). And similarly for the others.

Reply:
As for the EVENT_LIGHT_BUTTON_SHORT_UP idea: I tried to avoid renaming existing events to assure compatibility with existing code. Maybe @joeycastillo some time decides to take this road (of using different names) to clear things up.

Reply:
I agree it would cause some friction, but because in theory the faces are ending up in the same codebase and it wouldn't require significant changes to any people's existing work I think it's ok. But as you say, definitely @joeycastillo 's call :)